units/XNA0101/XNA0101_script.lua

```diff
@@ -1,20 +1,26 @@
--- T1 Air scout
-
 local NAirUnit = import('/lua/nomadsunits.lua').NAirUnit
 
+-- Tech 1 Air Scout
+---@class XNA0101 : NAirUnit
 XNA0101 = Class(NAirUnit) {
     DestructionPartsLowToss = {0},
     DestroySeconds = 7.5,
 
+    ---@param self XNA0101
+    ---@param builder Unit
+    ---@param layer string
     OnStopBeingBuilt = function(self,builder,layer)
         NAirUnit.OnStopBeingBuilt(self,builder,layer)
 
         self.LandingAnimManip = CreateAnimator(self)
         self.LandingAnimManip:SetPrecedence(0)
         self.Trash:Add(self.LandingAnimManip)
-        self.LandingAnimManip:PlayAnim(self:GetBlueprint().Display.AnimationLand):SetRate(1)
+        self.LandingAnimManip:PlayAnim(self.Blueprint.Display.AnimationLand):SetRate(1)
     end,
 
+    ---@param self XNA0101
+    ---@param new any
+    ---@param old any
```

Comment:
Another layer update by Jip from a while ago.
```suggestion
    ---@param new VerticalMovementState
    ---@param old VerticalMovementState
```